common/types/mapper/proto/admin.go

```diff
@@ -1324,7 +1319,7 @@ func ToAdminUpdateDomainIsolationGroupsRequest(t *adminv1.UpdateDomainIsolationG
 }
 
 func FromIsolationGroupConfig(in *types.IsolationGroupConfiguration) *apiv1.IsolationGroupConfiguration {
-	if in == nil {
+	if in == nil || *in == nil {
```

Comment:
nit: Do we need to distinguish between initialized and uninitialized maps? If no, `len(in)` might be more understandable.

Reply:
The way these tests are structured I'm trying to do an exact round trip.

Currently:
- in == nil -> nil
- *in == nil -> empty map
- len(*in) == 0 -> empty map

New behavior:
- in == nil -> nil
- *in == nil -> nil
- len(*in) == 0 -> empty map

For these specific mappers, since the parameter being passed in is a reference to the field on a request `in` is never actually nil. It'll just be pointing to a nil value. There are other invocations of this method that could actually be passing in nil though.